plugins/plugin_generate/bin-templates/js-template-runtime/src/app.js

```diff
@@ -15,6 +15,27 @@ var HelloWorldLayer = cc.Layer.extend({
         var mainscene = ccs.load(res.MainScene_json);
```

Comment:
res.MainScene_json is defined in 'plugins / plugin_generate / bin-templates / js-template-runtime / src / resource.js ' , and the resouce file is in the path' plugins / plugin_generate / bin-templates / js-template-runtime / res-landscape / res / ' or the path'plugins / plugin_generate / bin-templates / js-template-runtime / res-res-portrait  / res / '